packages/test/test-end-to-end-tests/src/test/gcTests/gcDataStoreRequests.spec.ts

```diff
@@ -17,7 +17,7 @@ import { requestFluidObject } from "@fluidframework/runtime-utils";
 import { ITestObjectProvider } from "@fluidframework/test-utils";
 import { describeNoCompat } from "@fluidframework/test-version-utils";
 import { IAckedSummary, IContainerRuntimeOptions, SummaryCollection } from "@fluidframework/container-runtime";
-import { flattenRuntimeOptions } from "./flattenRuntimeOptions";
```

Comment:
nit/suggestion: rename to `test/gc/...` instead of `test/gcTests/...` ?

Reply:
Renamed it.